### ISSUE TITLE
Upgrade from C126 -> C127 on Win x64 via 1.69.x - Nightly (Checking C127)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#39309]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.69.37 Chromium: 127.0.6533.17 (Official Build) nightly (64-bit)
-- | --
Revision | 27576aed3b77c509f77507cbb013f5d9a022135e
OS | Windows 11 Version 23H2 (Build 22631.3737)
```


<details>
<summary>C126 --> C127 on-demand upgrade check - PASSED</summary>

1. installed latest nightly `1.69.44 Chromium: 126.0.6478.114 (Official Build) nightly (64-bit)` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `1.69.44 Chromium: 127.0.6533.17 (Official Build) nightly (64-bit) ` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 6`**
----------------|----------------|----------------|---------------|------------
![image](https://github.com/brave/brave-browser/assets/98358127/bafcc894-8e21-4d23-a9bb-0643b5425007)|![image](https://github.com/brave/brave-browser/assets/98358127/bd578a9f-0842-4646-9820-807120fc5022)|![image](https://github.com/brave/brave-browser/assets/98358127/87343089-2529-47f6-ab55-9b894cc2b079)|![image](https://github.com/brave/brave-browser/assets/98358127/849e3566-87a9-4ec0-b177-2f86653eef5e)|![image](https://github.com/brave/brave-browser/assets/98358127/2482cfff-b4d4-4059-aece-fa329bbd36cb)
</details>

<details>
<summary>C126 --> C127 background upgrade check - `Pending`</summary>

1. installed latest nightly `1.69.40 Chromium 125.0.6422.76 (Official Build) nightly (64-bit)` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{10340040-BF4C-4EDF-92FF-BABE612FBE3F}` via `Task Scheduler`
5. waited ~30 min or so before launching Brave and ensured it was updated to `1.69.44 Chromium: 127.0.6533.17`  (removed `last checked` in `BraveSoftware\Update\` 
6. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|---------------|----------------|-----------
![image](https://github.com/brave/brave-browser/assets/98358127/526f60cc-85d3-4be0-8ef7-519185d4f84f)|![image](https://github.com/brave/brave-browser/assets/98358127/97c46a9f-f6f3-419c-880d-6b0738976018)|![image](https://github.com/brave/brave-browser/assets/98358127/c7f27f12-5fc2-44b3-836a-7abf8aeb5274)|![image](https://github.com/brave/brave-browser/assets/98358127/645a8b24-f2d1-4b84-a581-25accb0bb3b9)|![image](https://github.com/brave/brave-browser/assets/98358127/90d099a6-3f0f-4d05-a289-f19d8638c7ff)

</details>

<details>
<summary>C127 --> C127 on-demand upgrade check - PASSED</summary>

1. installed `1.69.37 Chromium: 127.0.6533.17 (Official Build) nightly (64-bit) ` 
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `1.69.44 Chromium: 127.0.6533.17` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|----------------|---------------|----------
![image](https://github.com/brave/brave-browser/assets/98358127/48864bd5-56fc-4dc1-850d-5c6feb664f22)|![image](https://github.com/brave/brave-browser/assets/98358127/00c1e038-f5c9-4e23-bfce-9422f30b2415)|![image](https://github.com/brave/brave-browser/assets/98358127/04999052-62a7-4a43-98d5-ed056ef57c5e)|![image](https://github.com/brave/brave-browser/assets/98358127/7865b157-96c6-4362-b472-18d3c6cd0941)|![image](https://github.com/brave/brave-browser/assets/98358127/a4803d1d-465e-472c-b0ea-0e9f9f8153b3)

</details>
<details>
<summary>C127 --> C127 background upgrade check - Pending</summary>

1. installed `1.68.38 Chromium: 126.0.6478.17` via `[BraveBrowserStandaloneNightlySetup_126_1_68_38.exe]` and closed Brave
2.  created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
3.  set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{9C318D44-9A78-4AFD-8B16-EC444A34098A}` via `Task Scheduler`
5. waited ~30min or so before launching Brave and ensured it was updated to `11.69.44 Chromium: 127.0.6533.17 `
6. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**  
----------------|----------------|------------|------------|--------- 



</details>

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.69.44 Chromium: 127.0.6533.17 (Official Build) nightly (32-bit)
-- | --
Revision | e92bcf2100d00816e00e55b886771eb3da69fdce
OS | Windows 10 Version 22H2 (Build 19045.4529)
```

<details>
<summary>C126 --> C127 on-demand upgrade check - PASSED</summary>

* installed `1.69.30 Chromium: 126.0.6478.71` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.69.44 Chromium: 127.0.6533.17` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/35fe7433-22fd-4752-93e8-5cd175bcdd60) | ![image](https://github.com/brave/brave-browser/assets/2602313/3cdbf1f2-137c-4b5f-af75-e0fa73393c18) | ![image](https://github.com/brave/brave-browser/assets/2602313/c4afdf24-c535-46a7-879d-f09e467ed92e) | ![image](https://github.com/brave/brave-browser/assets/2602313/99583c1a-0fef-463a-b793-94a8ec0059f4)
</details>

<details>
<summary>C127 --> C127 on-demand upgrade check - PASSED</summary>

* installed `1.69.37 Chromium: 127.0.6533.17` via `BraveBrowserStandaloneNightlySetup32_127_1_69_37.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.69.44 Chromium: 127.0.6533.17` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/b2217d19-a4cb-4949-97d1-ac73b8171f05) | ![image](https://github.com/brave/brave-browser/assets/2602313/bb57c15c-ff44-453c-878f-eb4e93e351ea) | ![image](https://github.com/brave/brave-browser/assets/2602313/64fd2330-3359-4a5e-b668-f23b0c7e89b1) | ![image](https://github.com/brave/brave-browser/assets/2602313/1c727ffb-d47f-4973-a6a4-1cd8530339e3)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[06/24/24 15:21:00.780][BraveUpdate:goopdate][10900:2252][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{3F5E27E5-7F57-4E7F-82F4-50D7CA097059}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[06/24/24 15:21:00.780][BraveUpdate:goopdate][10900:2252][Running installer][C:\Program Files\BraveSoftware\Update\Install\{3F5E27E5-7F57-4E7F-82F4-50D7CA097059}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[06/24/24 15:21:08.887][BraveUpdate:goopdate][10900:2252][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[06/24/24 15:21:08.887][BraveUpdate:goopdate][10900:2252][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[06/24/24 15:21:08.968][BraveUpdate:goopdate][10900:2252][Started process][1812]
[06/24/24 15:21:08.971][BraveUpdate:goopdate][10900:2252][LogEventHelper][4][12][1][[
Application update/install.
pid=10900, ver=1.3.361.151, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=127.1.69.37, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=127.1.69.37, Install result=0x00000000
]][]
[06/24/24 15:21:08.974][BraveCrashHandler][1812:1864][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[06/24/24 15:21:08.975][BraveCrashHandler][1812:1864][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[06/24/24 15:21:08.975][BraveCrashHandler][1812:1864][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[06/24/24 15:21:08.978][BraveCrashHandler][1812:1864][CrashHandler::RunUntilShutdown]
[06/24/24 15:21:09.075][BraveUpdate:goopdate][10900:10572][Started process][2052]
[06/24/24 15:21:09.076][BraveUpdate:goopdate][11524:4928][DllEntry exit][0x00000000]
[06/24/24 15:21:09.087][BraveUpdate:goopdate][2052:2436][DllEntry]["C:\Program Files\BraveSoftware\Update\BraveUpdate.exe" /ping PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48cmVxdWVzdCBwcm90b2NvbD0iMy4wIiB1cGRhdGVyPSJPbWFoYSIgdXBkYXRlcnZlcnNpb249IjEuMy4zNjEuMTUxIiBzaGVsbF92ZXJzaW9uPSIxLjMuMzYxLjE1MSIgaXNtYWNoaW5lPSIxIiBzZXNzaW9uaWQ9IntGRkU2NUUxNC1FMTIxLTQzMDgtODc3Qi1CMkUzRURDODYwNTN9IiB1c2VyaWQ9Ins4NEVGMDhENS1FOTY2LTRGOTYtQThCNy1BMUM3MUYxOEI2MDF9IiBpbnN0YWxsc291cmNlPSJ1cGRhdGUzd2ViLW9uZGVtYW5kIiB0ZXN0c291cmNlPSJhdXRvIiByZXF1ZXN0aWQ9InsyRjcwQkY0Qi03N0Q1LTQ4MzEtODRDMi1BNTUxRjkxODE1Mzd9IiBkZWR1cD0iY3IiIGRvbWFpbmpvaW5lZD0iMCI-PGh3IHBoeXNtZW1vcnk9IjIiIHNzZT0iMSIgc3NlMj0iMSIgc3NlMz0iMSIgc3NzZTM9IjEiIHNzZTQxPSIxIiBzc2U0Mj0iMSIgYXZ4PSIxIi8-PG9zIHBsYXRmb3JtPSJ3aW4iIHZlcnNpb249IjEwLjAuMTkwNDUuNDUyOSIgc3A9IiIgYXJjaD0ieDg2Ii8-PGFwcCBhcHBpZD0ie0M2Q0I5ODFFLURCMzAtNDg3Ni04NjM5LTEwOUY4OTMzNTgyQ30iIHZlcnNpb249IjEyNy4xLjY5LjM3IiBuZXh0dmVyc2lvbj0iMTI3LjEuNjkuNDQiIGFwPSJuaWdodGx5IiBsYW5nPSIiIGJyYW5kPSJHR0xTIiBjbGllbnQ9IiIgaW5zdGFsbGFnZT0iMCIgaW5zdGFsbGRhdGU9IjYzODQiPjxldmVudCBldmVudHR5cGU9IjEyIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTMiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiLz48ZXZlbnQgZXZlbnR0eXBlPSIxNCIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIgZG93bmxvYWRlcj0iYml0cyIgdXJsPSJodHRwczovL3VwZGF0ZXMtY2RuLmJyYXZlLmNvbS9kZWx0YS9CcmF2ZS1Ccm93c2VyLU5pZ2h0bHkvbmlnaHRseS93aW4vMTI3LjEuNjkuNDQvMTI3LjEuNjkuMzcveDg2L2JyYXZlX2luc3RhbGxlci1kZWx0YS1pYTMyLmV4ZSIgZG93bmxvYWRlZD0iNDM2MTc1MiIgdG90YWw9IjQzNjE3NTIiIGRvd25sb2FkX3RpbWVfbXM9IjEzOTAiLz48ZXZlbnQgZXZlbnR0eXBlPSIxNCIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIvPjxldmVudCBldmVudHR5cGU9IjE1IiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMyIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMTk2NjA5IiBzb3VyY2VfdXJsX2luZGV4PSIwIiB1cGRhdGVfY2hlY2tfdGltZV9tcz0iMzk1IiBkb3dubG9hZF90aW1lX21zPSIxNDczIiBkb3dubG9hZGVkPSI0MzYxNzUyIiB0b3RhbD0iNDM2MTc1MiIgaW5zdGFsbF90aW1lX21zPSI4MTA3Ii8-PC9hcHA-PC9yZXF1ZXN0Pg]
[06/24/24 15:21:09.088][BraveUpdate:goopdate][2052:2436][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[06/24/24 15:21:09.088][BraveUpdate:goopdate][2052:2436][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[06/24/24 15:21:09.088][BraveUpdate:goopdate][2052:2436][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[06/24/24 15:21:09.090][BraveUpdate:goopdate][2052:2436][C:\Program Files\BraveSoftware\Update\1.3.361.151\goopdate.dll][version 1.3.361.151][opt][dev]
[06/24/24 15:21:09.090][BraveUpdate:goopdate][2052:2436][is machine: 1]
[06/24/24 15:21:09.611][BraveUpdate:goopdate][2052:2436][DllEntry exit][0x00000000]
[06/24/24 15:21:09.611][BraveUpdate:goopdate][10900:10896][DllEntry exit][0x00000000]
```
</details>

<details>
<summary>C126 --> C127 background upgrade check - PASSED</summary>

* installed `1.69.30 Chromium: 126.0.6478.71` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{4145AD16-5F2B-4E52-AD4B-2CE797D1DB33}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.69.44 Chromium: 127.0.6533.17`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|-----------------
![image](https://github.com/brave/brave-browser/assets/2602313/e4fc2947-87db-4a66-8319-23231eacc9ba) | ![image](https://github.com/brave/brave-browser/assets/2602313/e49e9b51-7be8-4508-8a78-251ef73cbfca) | ![image](https://github.com/brave/brave-browser/assets/2602313/a6c7ee23-a7af-4fa6-85f9-948f3700cd8c) | ![image](https://github.com/brave/brave-browser/assets/2602313/638d4d3e-4cd6-423c-9998-72c03966df66)
</details>

<details>
<summary>C127 --> C127 background upgrade check - PASSED</summary>

* installed `1.69.37 Chromium: 127.0.6533.17` via `BraveBrowserStandaloneNightlySetup32_127_1_69_37.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{930DF1A1-991D-4459-81AA-E038181E0E7D}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.69.44 Chromium: 127.0.6533.17`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/edb081bd-1f8d-414b-9241-c5bb7dda5839) | ![image](https://github.com/brave/brave-browser/assets/2602313/cdd3e008-26ab-4696-a530-37840f8e94aa) | ![image](https://github.com/brave/brave-browser/assets/2602313/739c2025-7159-4df6-8d61-ef991602a9c7) | ![image](https://github.com/brave/brave-browser/assets/2602313/8229d705-23a9-45c8-b954-14a8b6bb7de1)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[06/24/24 16:53:17.599][BraveUpdate:goopdate][1648:11164][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{8850FF85-99E8-4859-88E2-6C9DFB02E420}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 0]
[06/24/24 16:53:17.599][BraveUpdate:goopdate][1648:11164][Running installer][C:\Program Files\BraveSoftware\Update\Install\{8850FF85-99E8-4859-88E2-6C9DFB02E420}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[06/24/24 16:53:24.727][BraveUpdate:goopdate][1648:11164][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[06/24/24 16:53:24.727][BraveUpdate:goopdate][1648:11164][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 2][]["C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe" --from-installer]
[06/24/24 16:53:24.790][BraveUpdate:goopdate][1648:11164][Started process][1840]
[06/24/24 16:53:24.790][BraveUpdate:goopdate][1648:11164][LogEventHelper][4][12][1][[
Application update/install.
pid=1648, ver=1.3.361.151, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=127.1.69.37, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=127.1.69.37, Install result=0x00000000
]][]
[06/24/24 16:53:24.790][BraveCrashHandler][1840:2056][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[06/24/24 16:53:24.790][BraveCrashHandler][1840:2056][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[06/24/24 16:53:24.790][BraveCrashHandler][1840:2056][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[06/24/24 16:53:24.805][BraveCrashHandler][1840:2056][CrashHandler::RunUntilShutdown]
[06/24/24 16:53:24.821][BraveUpdate:goopdate][4724:9256][DllEntry exit][0x00000000]
[06/24/24 16:53:24.821][BraveUpdate:goopdate][1648:7524][Started process][4304]
[06/24/24 16:53:24.837][BraveUpdate:goopdate][4304:4588][DllEntry]["C:\Program Files\BraveSoftware\Update\BraveUpdate.exe" /ping PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48cmVxdWVzdCBwcm90b2NvbD0iMy4wIiB1cGRhdGVyPSJPbWFoYSIgdXBkYXRlcnZlcnNpb249IjEuMy4zNjEuMTUxIiBzaGVsbF92ZXJzaW9uPSIxLjMuMzYxLjE1MSIgaXNtYWNoaW5lPSIxIiBzZXNzaW9uaWQ9IntGNkI1NUY2NS1FMjE2LTQ4RjYtOEE1RC1EQjYxNzkxNDA1MjN9IiB1c2VyaWQ9IntGNDhBNTA0OS0yMDQ2LTQ3QjQtQUVCQS04RUM5Qzg1MEJCRDZ9IiBpbnN0YWxsc291cmNlPSJzY2hlZHVsZXIiIHRlc3Rzb3VyY2U9ImF1dG8iIHJlcXVlc3RpZD0iezVERDlFRTU2LTQwN0EtNEVGNi04MkFDLUQyMzcxMENFODg4Nn0iIGRlZHVwPSJjciIgZG9tYWluam9pbmVkPSIwIj48aHcgcGh5c21lbW9yeT0iMiIgc3NlPSIxIiBzc2UyPSIxIiBzc2UzPSIxIiBzc3NlMz0iMSIgc3NlNDE9IjEiIHNzZTQyPSIxIiBhdng9IjEiLz48b3MgcGxhdGZvcm09IndpbiIgdmVyc2lvbj0iMTAuMC4xOTA0NS40NTI5IiBzcD0iIiBhcmNoPSJ4ODYiLz48YXBwIGFwcGlkPSJ7QzZDQjk4MUUtREIzMC00ODc2LTg2MzktMTA5Rjg5MzM1ODJDfSIgdmVyc2lvbj0iMTI3LjEuNjkuMzciIG5leHR2ZXJzaW9uPSIxMjcuMS42OS40NCIgYXA9Im5pZ2h0bHkiIGxhbmc9IiIgYnJhbmQ9IkdHTFMiIGNsaWVudD0iIiBpbnN0YWxsYWdlPSIwIj48ZXZlbnQgZXZlbnR0eXBlPSIxMiIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIvPjxldmVudCBldmVudHR5cGU9IjEzIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTQiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiIGRvd25sb2FkZXI9ImJpdHMiIHVybD0iaHR0cHM6Ly91cGRhdGVzLWNkbi5icmF2ZS5jb20vZGVsdGEvQnJhdmUtQnJvd3Nlci1OaWdodGx5L25pZ2h0bHkvd2luLzEyNy4xLjY5LjQ0LzEyNy4xLjY5LjM3L3g4Ni9icmF2ZV9pbnN0YWxsZXItZGVsdGEtaWEzMi5leGUiIGRvd25sb2FkZWQ9IjQzNjE3NTIiIHRvdGFsPSI0MzYxNzUyIiBkb3dubG9hZF90aW1lX21zPSIxMzY4MiIvPjxldmVudCBldmVudHR5cGU9IjE0IiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTUiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiLz48ZXZlbnQgZXZlbnR0eXBlPSIzIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIxOTY2MDkiIHNvdXJjZV91cmxfaW5kZXg9IjAiIHVwZGF0ZV9jaGVja190aW1lX21zPSI1OTQiIGRvd25sb2FkX3RpbWVfbXM9IjEzNzY1IiBkb3dubG9hZGVkPSI0MzYxNzUyIiB0b3RhbD0iNDM2MTc1MiIgaW5zdGFsbF90aW1lX21zPSI3MTI4Ii8-PC9hcHA-PC9yZXF1ZXN0Pg]
[06/24/24 16:53:24.837][BraveUpdate:goopdate][4304:4588][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[06/24/24 16:53:24.837][BraveUpdate:goopdate][4304:4588][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[06/24/24 16:53:24.837][BraveUpdate:goopdate][4304:4588][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[06/24/24 16:53:24.837][BraveUpdate:goopdate][4304:4588][C:\Program Files\BraveSoftware\Update\1.3.361.151\goopdate.dll][version 1.3.361.151][opt][dev]
[06/24/24 16:53:24.837][BraveUpdate:goopdate][4304:4588][is machine: 1]
[06/24/24 16:53:25.337][BraveUpdate:goopdate][4304:4588][DllEntry exit][0x00000000]
[06/24/24 16:53:25.337][BraveUpdate:goopdate][1648:12248][DllEntry exit][0x00000000]
```
</details>
